### DISCUSSION TITLE
How does MIB automatic install treat updates to B&O and similar amp systems?

### DISCUSSION THREAD

**Discussion by FrumpyMonk** [Mr-MIBonk/M.I.B._More-Incredible-Bash#520] [General]
I apologize if this information is found among the documentation for M.I.B. I've looked fairly extensively but couldn't find it.

Would someone please share with me how the "Standard" and/or automated install and updates to MIB2 units handle updating amps? 

I've not installed M.I.B. before, but I plan to do so soon on a 2018 SQ5. From what I've read in various discussion groups, when updating the firmware on MIB2 units the installer needs to be very careful NOT to update the B&O amp systems. So, how do the M.I.B. scripts handle that particular issue? Can I select a user-defined update (to make sure B&O is de-selected) or is it not updated by default?

Thank you for your response. 

Frumpy

**Comment by FrumpyMonk** [comment]
After some additional research, I believe that my original question is mute. That is, M.I.B. is a patch to the existing unit firmware; it does not update that firmware. If this understanding is correct, then M.I.B. doesn't perform the update to any amplifier system, and my original concern and question are not relevant. 

If I still misunderstand the difference between updating MMI firmware and installing More-Incredible-Bash, I hope someone will set me straight.

Thank you,

Frumpy.

**Comment by JamieP83** [comment]
how did it go?
im planning to update my 2018 SQ5 this week
